### PR TITLE
Upgrade branch

### PR DESCRIPTION
**Pull request by brandonsavage**
This is an upgrade to the modern playdoh layout.

### CODE REVIEW

**Comment by brandonsavage** [comment]
@peterbe r?

File: project/dll/static/dll/main.css [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+.page-header input {
+    top: 20px;
+    right: 20px;
+    position: absolute;
+    border-radius: 0.5em;
+	-webkit-border-radius: 0.5em;
+	-moz-border-radius: 0.5em;
+	border: 1px solid #A0A0A0;
+	width: 240px;
+	background: url("/media/img/dll/search.png") no-repeat scroll 5px center #FFFFFF;
```
Comment (peterbe) [inline project/dll/static/dll/main.css]: Are you sure this works? That supposed to be `/static/`. 

"media" is where you put things like user uploaded images or files. Static is what comes with the source code. 

Perhaps this does work, but it's wrong and worth doing something about.

File: project/settings/local.py [hunk 2]
```diff
@@ -0,0 +1,61 @@
+# This is an example settings_local.py file.
```
Comment (peterbe) [inline project/settings/local.py]: This file should NOT be checked in. `settings/local.py-dist` should but not this one. `settings/local.py` should be in your `.gitignore`

File: project/settings/local.py [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+# Copy it and add your local settings here.
+
+from . import base
+
+DATABASES = {
+    'default': {
+        'ENGINE': 'django.db.backends.mysql',
+        'NAME': 'dlldir',
+        'USER': 'root',
+        'PASSWORD': 'MySQLR00+',
```
Comment (peterbe) [inline project/settings/local.py]: Interesting! 
Do you see now why this file should not be checked in. 

If that's a password used on Mozilla servers you need to file a rather urgent bug to have this reset.

File: static/js/dll/detail.js [hunk 4]
```diff
@@ -0,0 +1,18 @@
+$(document).ready(function () {
+   
+   $('ul.tab-list li a').click(function() {
+       classAttr = $(this).attr('class');
```
Comment (peterbe) [inline static/js/dll/detail.js]: what if it has multiple classes?

File: static/js/dll/detail.js [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+       $('ul.tab-list li').removeClass('active')
```
Comment (peterbe) [inline static/js/dll/detail.js]: Inconsistent use of semi-colons.

File: vendor [hunk 6]
```diff
@@ -1 +1 @@
-Subproject commit 787b0bd7ad552a7c2b2a4f65ed37422d652082c8
+Subproject commit 7040835e4bff630df108ce2d1acbabe6a5e156ba
```
Comment (peterbe) [inline vendor]: This one is 3 months old. And it points to django 1.4.2 which is security-wise out of date.

**Comment by peterbe** [comment]
You have 2 very out-of-date jquery files in there.

**Comment by peterbe** [comment]
r-
- All the security stuff with local.py and old playdoh-lib.
- The word "project" is supposed to be replaced with something more relevant like "dragnet". `mv project/ dragnet/; git grep -n project`

File: static/js/dll/detail.js (hunk 4 above)
Comment (brandonsavage) [inline static/js/dll/detail.js]: Then the javascript will need to change. In this case, I designed it to have only one class.

Comment (peterbe) [inline static/js/dll/detail.js]: Looks fragile to me. Is there no other way?

**Comment by brandonsavage** [comment]
I have made the requested changes. @peterbe, r?

**Comment by peterbe** [comment]
Sigh on premature rebasing. Now I can't see what's changed since. But I'll try to go over it again.

File: setup.py [hunk 7]
```diff
@@ -0,0 +1,18 @@
+import os
```
Comment (peterbe) [inline setup.py]: Does this file even work? `python setup.py build` will test that. 
If you're going to use it, why not fill it in entirely?

File: static/examples/main.css [hunk 8]
```diff
@@ -0,0 +1,52 @@
+/* An example CSS file that will be minified. */
```
Comment (peterbe) [inline static/examples/main.css]: Surely "examples" is from the playdoh days. Rm.

**Comment by peterbe** [comment]
r+ with nits
- remove the extra jquery-1.9
- remove stuff from "examples"
- if you're not using browserid, remove it

I'll leave it up to you to rebase and merge.

**Pull request merged**